app/exporters/patients_exporter.rb

```diff
@@ -116,9 +120,18 @@ def self.csv_fields(patient)
       patient.id,
       *medications_for(patient)
     ]
+
+    csv_fields.insert(zone_column_index, patient.address.zone) if zone_column_index
+    csv_fields
   end
 
   def self.medications_for(patient)
     patient.prescription_drugs.flat_map { |drug| [drug.name, drug.dosage] }
   end
+
+  private
+
+  def self.zone_column
+    "Patient #{Address.human_attribute_name :zone}"
```

Comment:
Do we need to jump through AR hoops because "zone" is called different things in different countries?

Reply:
Yeah, actually we would be doing this for all the columns but it doesn't matter there really.